homeassistant/components/unifiprotect/config_flow.py

```diff
@@ -365,33 +367,53 @@ async def async_step_init(
         self, user_input: dict[str, Any] | None = None
     ) -> FlowResult:
         """Manage the options."""
+
+        values = user_input or self.config_entry.options
+        schema = vol.Schema(
+            {
+                vol.Optional(
+                    CONF_DISABLE_RTSP,
+                    description={
+                        "suggested_value": values.get(CONF_DISABLE_RTSP, False)
+                    },
+                ): bool,
+                vol.Optional(
+                    CONF_ALL_UPDATES,
+                    description={
+                        "suggested_value": values.get(CONF_ALL_UPDATES, False)
+                    },
+                ): bool,
+                vol.Optional(
+                    CONF_OVERRIDE_CHOST,
+                    description={
+                        "suggested_value": values.get(CONF_OVERRIDE_CHOST, False)
+                    },
+                ): bool,
+                vol.Optional(
+                    CONF_MAX_MEDIA,
+                    description={
+                        "suggested_value": values.get(CONF_MAX_MEDIA, DEFAULT_MAX_MEDIA)
+                    },
+                ): vol.All(vol.Coerce(int), vol.Range(min=100, max=10000)),
+                vol.Optional(
+                    CONF_IGNORED,
```

Comment:
What's the difference between this option and being able to disable a device which the user can already do?

Reply:
Unless I am missing something, you cannot disable devices only entities. I see some code referencing in device registry about disabling devices, but I cannot find anywhere in the UI that allows it or integration that does it.

Reply:
It's available under the device name edit.

Reply:
So, it is. Really inconsistent UX that Disable/Delete are in different places across the board. 

That would make this PR completely redundant then.

Reply:
> That would make this PR completely redundant then.

Probably makes sense to revert it now before it goes out to 2022.9.x and then we don't have a breaking change later.

Reply:
I was thinking there was a reason we needed to ignore it from coming back though.

Reply:
Never-mind, that was from an earlier iteration.  Assuming there is agreement, I think we are all good with a revert and the prescribed method should be to disable the device if they don't want to adopt it.